src/Bridges/Constraint/geomean.jl

```diff
@@ -40,26 +40,48 @@ Now, this is equivalent to
 """
 struct GeoMeanBridge{T, F, G, H} <: AbstractBridge
     # Initially, (t, x) is of dimension d so x is dimension (d-1)
-    # We create n new variables so that there is 2^l = d-1+n variables x_i
+    # TODO the sentence below is a little confusing, the n isn't used anywhere
+    # We create n new variables so that there are 2^l = d-1+n variables x_i
     # We then need to create 2^l-1 new variables (1+2+...+2^{l-1})
+    # So the total number of variables is d+2^l-1
     d::Int
     xij::Vector{MOI.VariableIndex}
     tubc::CI{F, MOI.LessThan{T}}
     socrc::Vector{CI{G, MOI.RotatedSecondOrderCone}}
+    nonneg::Union{Nothing, CI{G, MOI.Nonnegatives}}
 end
 function bridge_constraint(::Type{GeoMeanBridge{T, F, G, H}}, model,
                            f::MOI.AbstractVectorFunction,
                            s::MOI.GeometricMeanCone) where {T, F, G, H}
     d = s.dimension
+    if d == 1
+        # TODO change to a standard error
+        error("Dimension of GeometricMeanCone must be greater than 1.")
+    end
     n = d - 1
     l = ilog2(n)
     N = 1 << l
-    xij = MOI.add_variables(model, N - 1)
+    socrc = Vector{CI{G, MOI.RotatedSecondOrderCone}}(undef, N - 1)
     f_scalars = MOIU.eachscalar(f)
+    t = f_scalars[1]
+    SG = MOIU.scalar_type(G)
+
+    if d == 2
+        # TODO better to use nothing?
+        xij = MOI.VariableIndex[]
+        tubc = MOIU.normalize_and_add_constraint(
+            model, MOIU.operate!(+, T, t, -1.0 * f_scalars[2]), MOI.LessThan(zero(T)),
+            allow_modify_function=true)
+        # TODO hacky, what is the correct way to get [x] the correct type?
```

Comment:
`nonneg` don't have to be a `G`-in-`Nonnegatives`, it can be another function type than `G`. In this case, it seems to be `H`-in-`Nonnegatives`.